### PR TITLE
Add CuPy core devs as maintainer 

### PR DESCRIPTION
**Pull request by leofang**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

As per requested in https://github.com/cupy/cupy/issues/2274#issuecomment-547273996. cc: @jakirkham, @kmaehashi, @pentschev

### CODE REVIEW

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```
[the remark this answers is not in the document]
Comment (leofang) [reply recipe/meta.yaml]:
```suggestion
  number: 0
```

[the remark this answers is not in the document]
Comment (jakirkham) [reply recipe/meta.yaml]: Thanks!

**Comment by kmaehashi** [comment]
Thank you! Could you add @beam2d,@asi1024,@emcastillo,@hvy,@niboshi,@toslunar as well?

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
You should all have gotten GitHub invites to join conda-forge (specifically on the `@conda-forge/cupy` team), which will confer write permissions on this feedstock. More details in [this comment]( https://github.com/conda-forge/cupy-feedstock/commit/8358fc32514b89ed85735fa67d883c3f487a998a#commitcomment-35751919 ). Please let us know if you have any questions and welcome to conda-forge! 😄

**Pull request merged**